assets/sass/components/_Logo.scss

```diff
@@ -1,17 +1,33 @@
+
 /*
  * Logo
- * 
+ *
  * This component styles the Logo(partials/logo.html).
  */
-g.Logo {
+g.MainLogo {
   stroke: $logoColor;
 }
 
 /*
  * LogoLink
- * 
+ *
  * This component can be used to wrap the component into an anchor tag.
  */
-a.LogoLink {
+a.MainLogoLink {
   line-height: 0px;
 }
+
+
+.Logo{
+  max-width: 50vw;
```

Comment:
The Logo size was perfectly fine [https://www.quora.com/What-is-the-best-icon-size](https://www.quora.com/What-is-the-best-icon-size)

Reply:
MainLogo vs. UniHackLogo...

Reply:
I'm not trying to change the size of the UniHackLogo, I just want the sponsors and organizers logos to be displayed properly on desktop and mobile.

Naming of the classes will be changed.